FAQ.md

```diff
@@ -0,0 +1,66 @@
+# FAQ
+
+Answers to frequently asked questions about the [cleanvision](https://github.com/cleanlab/cleanvision) open-source package.
+
+The code snippets in this FAQ come from a fully executable notebook you can run via Colab or locally by downloading it [here](https://github.com/cleanlab/cleanlab/blob/master/docs/source/tutorials/faq.ipynb).
```

Comment:
not sure what is going on here, looks this is just copy/pasted with no proofreading

Reply:
sorry about that, did that in a haste. Removing it